zqd/packet/packet.go

```diff
@@ -193,29 +208,36 @@ func (p *IngestProcess) writeData(ctx context.Context) error {
 	// convert logs into sorted bzng
 	zr, err := scanner.OpenFiles(resolver.NewContext(), files...)
 	if err != nil {
-		return err
+		return nano.Ts(0), nano.Ts(0), err
 	}
 	defer zr.Close()
 	// For the time being, this endpoint will overwrite any underlying data.
 	// In order to get rid errors on any concurrent searches on this space,
 	// write bzng to a temp file and rename on successful conversion.
 	bzngfile, err := p.space.CreateFile("all.bzng.tmp")
 	if err != nil {
-		return err
+		return nano.Ts(0), nano.Ts(0), err
 	}
 	zw := bzngio.NewWriter(bzngfile)
-	const program = "sort -limit 10000000 ts"
-	if err := search.Copy(ctx, zw, zr, program); err != nil {
+	const program = "sort -limit 10000000 ts | (filter *; head 1; tail 1)"
```

Comment:
I would suggest writing the program like this: `sort -limit 10000000 ts; min(ts) max(ts)` and have the second output to second file called `all.spancache.bzng` adjacent to`all.bzng`. We wouldn't need to have the logic in `space` to create the `info.json` file.

Reply:
After offline discussion, we agreed to go with json.